docs/organization-integration/onpremise-integration.md

```diff
@@ -0,0 +1,173 @@
+# Sigrid on-premise integration
+
+Most of this documentation refers to the software-as-a-service version of Sigrid, which can be accessed via [sigrid-says.com](https://sigrid-says.com) and is used by the large majority of Sigrid users. However, SIG also offers an on-premise version of Sigrid. This document covers everything you need to integrate Sigrid on-premise in your environment. It also covers the functional differences between the SaaS version and the on-premise version, though these differences are relative minor.
+
+<sig-toc></sig-toc>
+
+## High-level overview
+
+<img src="../images/onpremise-overview.png" width="550" />
+
+- Sigrid on-premise is based on [Docker containers](https://en.wikipedia.org/wiki/Docker_%28software%29). There are two types of containers:
+  - Application containers that should be deployed in a [Kubernetes](https://en.wikipedia.org/wiki/Kubernetes) cluster, based on a [Helm chart](https://helm.sh) that is provided by SIG.
+  - Analysis containers that run from a build pipeline within your development environment (e.g. GitHub or GitLab).
+- SIG provides the necessary images through a container registry. The section [obtaining Sigrid on-premise](#obtaining-sigrid-on-premise) contains more information on how you can obtain and update these Docker containers.
+- Authentication is based on your identity provider, using [OpenID Connect](https://openid.net/developers/how-connect-works/). 
+- Analyses are triggered from a build pipeline. The analysis results are then imported into a Postgres database, so they can be viewed in Sigrid.
+- Large files are stored in an [S3-compatible object store](https://aws.amazon.com/s3/).
+
+Some Sigrid on-premise features are *optional* based on your environment:
+
+- The Open Source Health feature requires outbound internet access, to retrieve the latest version of all known vulnerabilities in open source libraries. 
+- When viewing detailed analysis results, Sigrid displays relevant source code files within Sigrid. For this to work, a web-accessible code storage needs to be available. This integrates with Sigrid via [OAuth](https://oauth.net/2/). For this to work, the identity provider used for Sigrid authentication and for the code storage needs to be the same.
+
+### Environment requirements
```

Comment:
Do we want to specify versions? For example we have a minimum version requirement of kubernetes version 1.21  and Postgres version 13 but only aim to support non EOL versions of kubernetes (the last three releases)?